app/java/src/main/java/com/google/cloudclientapi/EnvironmentVars.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloudclientapi;
+
+/**
+ * This class is used for obtaining values of environment variables. Yes,
+ * System.getenv().get("MY_ENV_VAR") is simple enough already, but java.lang.System cannot be mocked
+ * for unit tests.
+ */
+public class EnvironmentVars {
+  public static String get(String key) {
+    return System.getenv().get(key);
+  }
+}
```

Comment:
I had to introduce this EnvironmentVars class because you cannot mock the `System.getenv` function.